src/hooks/twilio/useVideoRoomState.tsx

```diff
@@ -144,7 +122,7 @@ export const useVideoRoomState = (
   );
 
   return {
-    loading: participantsLoading || roomLoading,
+    loading: status === VideoCommsStatus.Connecting || roomLoading,
```

Comment:
might be good if `useVideoRoomState` returns the uniforming `LoadStatus` fields

Reply:
Agreed - for now I'm minimising the scope of this changeset